dashboard/test/ui/step_definitions/eyes_steps.rb

```diff
@@ -37,7 +37,8 @@
 
 And(/^I close my eyes$/) do
   @browser = @original_browser
-  @eyes.close
+  fail_on_mismatch = !CDO.ignore_eyes_mismatches
+  @eyes.close(fail_on_mismatch)
```

Comment:
For future reference: [Here's the implementation of `Applitools::Eyes.close`](https://github.com/applitools/Eyes.Selenium.Ruby/blob/master/lib/applitools/eyes.rb#L237) with its one argument `raise_ex`.